src/string.cr

```diff
@@ -721,31 +721,44 @@ class String
   # as character indices. Indices can be negative to start
   # counting from the end of the string.
   #
-  # Raises `IndexError` if the range's start is not in range.
+  # Raises `IndexError` if the range's start is out of range.
   #
   # ```
-  # "hello"[0..2]   # "hel"
-  # "hello"[0...2]  # "he"
-  # "hello"[1..-1]  # "ello"
-  # "hello"[1...-1] # "ell"
+  # "hello"[0..2]   # => "hel"
+  # "hello"[0...2]  # => "he"
+  # "hello"[1..-1]  # => "ello"
+  # "hello"[1...-1] # => "ell"
+  # "hello"[6..7]?  # raise IndexErrors
```

Comment:
```suggestion
  # "hello"[6..7]?  # raises IndexError
```

Reply:
```suggestion
  # "hello"[6..7]   # raises `IndexError`
```

Reply:
@r00ster91 surprisingly not, grep the code to see precedents.

Reply:
Deleted that suggestion, noticed it's in code so surrounding `IndexError` by grave accents wouldn't be useful.